### DISCUSSION TITLE
Is there a way to add a second gateway

### DISCUSSION THREAD

**Discussion by str8edgedave** [netalertx/NetAlertX#609] [Q&A]
I have two external I addresses for my home lan. There are separate firewalls for both of them. Is there a way to configure Pi.Alert to test internet connectivity to both?

**Comment by jokob-sk** [comment]
Hey @str8edgedave ,

It's not currently configurable, however there is a Website monitor plugin. I'm wondering, if it would be possible to add the IP address of your second gateway in there and monitor it that way? I haven't tested this, just an idea to try out. 

![image](https://github.com/jokob-sk/Pi.Alert/assets/96159884/d8556077-8463-4733-8b49-e5cace5f3be8)

**Reply by str8edgedave** [reply]
Thanks for responding. I'll try it out this evening.  The network is divided into home dev and home prod. I'd like to use one instance to monitor both sides.
